manifest.json

```diff
@@ -12,19 +12,19 @@
     },
```

Comment:
Signature for `...caUq`

`Er9gQtWlgth/bHPVVlX7wsr6Zf3Pjjwuz561xjW9DAvNeLcc4uSit5u4qy33kgth5BiW7iVsxcItnskpmY2HDg==`